irispy/iris_tools.py

```diff
@@ -21,6 +21,9 @@
 from sunpy.util.net import check_download_file
 from ndcube import NDCube
 
+from astropy.time import Time
+from IPython.core import debugger ; debug = debugger.Pdb().set_trace
```

Comment:
Delete this import.

Reply:
Yup, deleted `from IPython.core import debugger ; debug = debugger.Pdb().set_trace`.